### PR TITLE
Adding Export to World Weather Record dialog 

### PR DESCRIPTION
**Pull request by HawardKetoyoMsatsi**
Added two blank forms dlgExportToWWR and sdgExportToWWR. This solves issue #6110
@africanmathsinitiative/developers and @rdstern this is ready for review. 
It contains frmmain changes therefore it should be merged as soon as possible.

### CODE REVIEW

**Comment by dannyparsons** [comment]
Thanks for this. Are the menu items hidden? It's fine to add them to frmMain but they should be hidden until the dialogs are implemented. Alternatively, you could have created the dialog but not added the menu items yet. Either is ok.

Also, it's good to reference the issue, but you don't want this pull request to close the issue, because that will be when you actually do the dialog. In your next PR when you implement the dialog, you can use "fixes #xxx" to reference the issue and close it when merged.

**Comment by HawardKetoyoMsatsi** [comment]
Well noted, I have disabled the menu.

File: instat/frmMain.resx [hunk 1]
```diff
@@ -1058,6 +1058,15 @@
   <data name="mnuClimaticFileExportToCPT.Text" xml:space="preserve">
     <value>Export to CPT...</value>
   </data>
+  <data name="ExportToWWRToolStripMenuItem.Enabled" type="System.Boolean, mscorlib">
+    <value>False</value>
```
Comment (lloyddewit) [inline instat/frmMain.resx:1062]: Should both the `Enabled` **and** `Visible` properties be set to false? (we don't want the user to see a greyed-out menu item, we prefer the user not to see the menu item at all)

File: instat/frmMain.resx [hunk 2]
```diff
@@ -1058,6 +1058,15 @@
...
+  </data>
+  <data name="ExportToWWRToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
```
Comment (lloyddewit) [inline instat/frmMain.resx:1064]: I'm not familiar with the `resx` naming convention but should this new menu item be called `mnuExportToWWRToolStrip`?
Please could you check with one of the other developers? thanks

File: instat/frmMain.resx [hunk 3]
```diff
@@ -3544,31 +3553,31 @@
     <value>Wind/Pollution Rose...</value>
   </data>
   <data name="mnuStructuredCircularOtherRosePlotsPercentileRose.Size" type="System.Drawing.Size, System.Drawing">
-    <value>180, 22</value>
+    <value>168, 22</value>
```
Comment (lloyddewit) [inline instat/frmMain.resx:3556]: Do you know why these Rose plot values changed (I assume they aren't connected to this PR)?

**Review by lloyddewit — CHANGES_REQUESTED** [review]

File: instat/frmMain.resx (hunk 2 above)
Comment (dannyparsons) [reply instat/frmMain.resx:1064]: Yes `mnuExportToWWRToolStrip` would be a sensible name.

File: instat/frmMain.resx (hunk 3 above)
Comment (dannyparsons) [reply instat/frmMain.resx:3556]: It does happen automatically sometimes, I am not really sure why. If it doesn't look any different in the design then it should be ok.

**Comment by dannyparsons** [comment]
@HawardKetoyoMsatsi while you are here, could you also change the label for Climatic > Prepare > SPI to "SPI/SPEI" if Roger agrees. I will update the dialog itself.

**Comment by rdstern** [comment]
Agreed.  Good idea to include SPEI.  I assume we can include our own evapotranspiration.  I did check with the package people if we could use their routine, but it seems to be limited to monthly data.

File: instat/frmMain.resx (hunk 2 above)
Comment (HawardKetoyoMsatsi) [reply instat/frmMain.resx]: I have fixed naming of the menu item mnuExportToWWRToolStripMenuItem .

**Comment by HawardKetoyoMsatsi** [comment]
Since Roger agreed I have left the dialog for @dannyparsons to update the last suggestion raised concerning this dialog.

File: instat/frmMain.resx (hunk 1 above)
Comment (HawardKetoyoMsatsi) [reply instat/frmMain.resx]: I have set both enabled and visible properties to false

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern I approved from my side but it still requires an update from @dannyparsons

**Review by dannyparsons — APPROVED** [review]

**Pull request merged**